### ISSUE TITLE
Log accessed resources with it's itsyou.online username

### ISSUE THREAD

**Issue by robvanmieghem** [itsyouonline/caddy-integration#2]

**Comment by iwanbk** [comment]
implementation can be found in `oauth` branch.

log example:
```
::1 [12/Jan/2017:09:52:50 +0700] "GET /data/a.txt HTTP/1.1" ibk
```

format was modified from caddy standard log format, with these changes:
- remove last 2 fields : http status code and http response size
- add IYO username as last field

**Comment by iwanbk** [comment]
implemented in #3.

Use `fmt.Printf` instead of `log` package because:
- we only need to print to stdout
- `caddy` might modify `log` package output device
